buildSrc/src/main/kotlin/io.opentelemetry.instrumentation.base.gradle.kts

```diff
@@ -1,7 +1,16 @@
 /** Common setup for manual instrumentation of libraries and javaagent instrumentation. */
 
 plugins {
-  id("otel.java-conventions")
+  `java-library`
+}
+
+afterEvaluate {
```

Comment:
Compared to previous version, this publishable one removes the (javaagent, library, testing) folder scheme handling from this file (moved to otel.java-conventions) and adds this application of the BOM